client/dashboard/index.js

```diff
@@ -37,16 +33,4 @@ class Dashboard extends Component {
 	}
 }
 
-const onboardingData = getSetting( 'onboarding', {} );
-
-export default compose(
-	!! onboardingData.tasksStatus
-		? withOnboardingHydration( { tasksStatus: onboardingData.tasksStatus } )
-		: identity,
-	withSelect( ( select ) => {
-		const { getProfileItems } = select( ONBOARDING_STORE_NAME );
-		const profileItems = getProfileItems();
-
-		return { profileItems };
-	} )
-)( Dashboard );
+export default Dashboard;
```

Comment:
The query/path are passed into this component so all this stuff was completely unused.